samples/client/others/typescript-angular/builds/composed-schemas-tagged-unions/model/catComposed.ts

```diff
@@ -12,7 +12,9 @@
 
 
 export interface CatComposedModel { 
+    /** */
     petType: string;
+    /** */
```

Comment:
instead of having lines with empty comments, what about defaulting to the property name if the description is empty?

Reply:
I don't mind adding that, I did that on my first attempt but it looked a bit messier to be honest since it's redundant information. The property name is clearly defined below the comment, and there is no "@property" tag or anything to be used for this purpose. I think the common thing to put in the do block is tags and description.